docs/DeveloperGuide.md

```diff
@@ -452,6 +447,145 @@ facilitates the lending of money and ensures data integrity by validating input
 The TransactionList class provides essential functionalities for managing a list of transactions.
 Its methods facilitate the addition, removal, editing, and retrieval of transactions, ensuring efficient management of transactions within a group.
 
+### DateTime
+
+<ins> Overview </ins>
+
+The DateTime class handles all operations in LongAh involving the tracking of time. This includes storing and printing
+the datetime elements in dated transactions, parsing user's date & time related inputs as well as filtering transactions
+according to their stored date & time. Implementation of the class is made possible with the help of the *java.time*
+system class.
+
+<ins> Class Fields </ins>
```

Comment:
Iirc we standardise to "Class Structure" instead.

Reply:
Resolved!